lpa-submissions-e2e-tests/cypress/support/commands.js

```diff
@@ -53,3 +53,7 @@ Cypress.Commands.add('clickBackButton', require('./common/clickBackButton'));
 Cypress.Commands.add('validateErrorMessage', require('./common/validateErrorMessage'));
 
 Cypress.Commands.add('completeQuestionnaire', require('./common/completeQuestionnaire'));
+
+Cypress.Commands.add('hasLink', require('./common/hasLink'));
+
+Cypress.Commands.add('clickDataCyElement', require('./common/clickDataCyElement'));
```

Comment:
I don't know the answer to this, but I ask myself this Q continually when using Cypress:

https://docs.cypress.io/api/cypress-api/custom-commands#1-Don-t-make-everything-a-custom-command

Could this be a function inside the integration layer, rather than extracted to support?